### DISCUSSION TITLE
Info regarding image size

### DISCUSSION THREAD

**Discussion by Sarath-C-Koppolu** [ultralytics/ultralytics#13376] [General]
Hello,
Thanks for your reply. 
I have updated the code and start running with cross validation folds. 

Now earlier i needed to train with 5 different image sizes. 
Now I don't have much time.. 

I have been suggested that training with 1280 image size and we can validate using Val with different image sizes... 

So if I run with 1280 image size and can evaluate for 320,480,640,800,960, image sizes !!

Can this happen if so 
How do I change my val ?

And since the folds only create train and val I have saved the folder test in each fold and added test in data.yaml . 
Will this work if I use split= test? 

If not how do I change my folders and yaml so split works as I need map scores of test set

**Comment by glenn-jocher** [comment]
Hello,

Thank you for reaching out with your questions!

1. **Training and Validation with Different Image Sizes:** Yes, you can train your model with an image size of 1280 and validate it with different image sizes. To do this, you can specify the `imgsz` parameter during validation for each desired size. For example:
   ```bash
   yolo val model=yolov8n.pt imgsz=320
   ```
   Repeat this for each image size (320, 480, 640, 800, 960) you want to validate.

2. **Using a Test Split in Your Data:** If you've added a test folder and updated your `data.yaml` to include a test path, using `split=test` should work as expected. Ensure your `data.yaml` includes something like:
   ```yaml
   train: path/to/train
   val: path/to/val
   test: path/to/test
   ```
   Then, you can validate on your test set by specifying:
   ```bash
   yolo val model=path/to/model.pt data=data.yaml split=test
   ```

This setup should allow you to obtain mAP scores for your test set as needed. If you encounter any issues, please ensure your paths are correctly specified in your `data.yaml`.

Best of luck with your training and validation! Let us know if you have any more questions.
